metadata.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import os
+
+# Path of .csv file
+input_path = input("Enter path of .csv metadata file: ")
+output_dir = input("Enter directory for output (leave blank for current directory): ")
```

Comment:
It may be better to drive the script from command line with `--input_path` and `--output_dir` as arguments and use `argparse` to process these inputs. An [example tutorial](https://levelup.gitconnected.com/the-easy-guide-to-python-command-line-arguments-96b4607baea1).

If you need any help, please poke me!